challenge_receptionist/src/challenge_receptionist/learn_guest.py

```diff
@@ -134,14 +133,10 @@ def __init__(self, robot,
             else:
                 smach.StateMachine.add('HEAR_DRINK_ANSWER',
                                        HearOptionsExtraPicoVoice(
-                                           robot, "receptionist", answer.writeable, "drinks", look_at_standing_person=True),
-                                       transitions={'heard': 'PROCESS_ANSWER',
+                                           robot, "receptionist", guest_drink_des.writeable, ["drinks"], look_at_standing_person=True),
```

Comment:
This argument needs to be a designator resolving to a HMIResult......

Reply:
This argument is already a designator resolving to a HMIResult coming from HandleSingleGuest - receptionist.py